lib/mjsonwp/mjsonwp.js

```diff
@@ -245,6 +245,11 @@ function buildHandler (app, method, path, spec, driver, isSessCmd) {
         driverRes = null;
       }
 
+      // if the status is not 0,  throw the appropriate error for status code.
+      if (util.hasValue(driverRes) && util.hasValue(driverRes.status) && parseInt(driverRes.status, 10) !== 0) {
+        throw errorFromCode(driverRes.status, driverRes.value);
```

Comment:
hmm, to me this means we should fix the other drivers ASAP rather than add workarounds here. there should be two basic cases:

1) we call a driver command and it either returns a value or throws
2) we proxy to a JSONWP-based server, and the jsonwp-proxy code converts that into a return value or a throw

in no case should we try to do handle some other kind of response here IMO